private/buf/bufgen/internal/bufgenv2/file_option.go

```diff
@@ -24,42 +24,42 @@ import (
 )
 
 const (
-	// fileOptionJavaPackage is the file option java_package.
```

Comment:
Occurrences of `fileOptionXyz` -> `FileOptionXyz` are the only changes in this file